src/test/java/syncsquad/teamsync/viewmodel/MainViewModelTest.java

```diff
@@ -47,6 +48,7 @@ public void setUp() {
         // Configure logic mock
         when(logic.getFilteredPersonList()).thenReturn(personList);
         when(logic.getMeetingList()).thenReturn(meetingList);
+        when(logic.getCurrentWeek()).thenReturn(new TimetableWeek());
```

Comment:
Avoid creating directly, expose timetable week as a private var in line 38, set up mock currentweek at line 47, then reference the timetable week private var in line 51.